stdlib/universe/kaufmansAMA_test.go

```diff
@@ -0,0 +1,209 @@
+package universe_test
+
+import (
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/execute/executetest"
+	"github.com/influxdata/flux/querytest"
+	"github.com/influxdata/flux/stdlib/universe"
+	"testing"
+)
+
+func TestKamaOperation_Marshaling(t *testing.T) {
+	data := []byte(`{"id":"kaufmansAMA","kind":"kaufmansAMA","spec":{"n":1}}`)
+	op := &flux.Operation{
+		ID: "kaufmansAMA",
+		Spec: &universe.KamaOpSpec{
+			N: 1,
+		},
+	}
+	querytest.OperationMarshalingTestHelper(t, data, op)
+}
+
+func TestKama_PassThrough(t *testing.T) {
+	executetest.TransformationPassThroughTestHelper(t, func(d execute.Dataset, c execute.TableBuilderCache) execute.Transformation {
+		s := universe.NewkamaTransformation(
+			d,
+			c,
+			&universe.KamaProcedureSpec{},
```

Comment:
So I guess that kama will just forward its input tables to its output if `n` is zero and `columns` is empty... Do both of these conditions have to be true for kama to be a passthrough?

Do we have other functions that behave this way, i.e., they become pass through functions for certain parameter values.  I could imagine returning an error for cases like this as well.

Reply:
Looking over code in the `Process` method, it seems like `columns:[]` should be a pass through, but there should be an error returned if `n` is not positive.